src/EncryptedFieldValueManagerInterface.php

```diff
@@ -61,6 +61,23 @@ public function saveEncryptedFieldValues(ContentEntityInterface $entity);
   public function getEncryptedFieldValue(ContentEntityInterface $entity, $field_name, $delta, $property);
 
   /**
+   * Loads an existing EncryptedFieldValue entity.
+   *
+   * @param \Drupal\Core\Entity\ContentEntityInterface $entity
+   *   The entity to check.
+   * @param string $field_name
+   *   The field name to check.
+   * @param int $delta
+   *   The field delta to check.
+   * @param string $property
+   *   The field property to check.
+   *
+   * @return bool|\Drupal\field_encrypt\Entity\EncryptedFieldValue
+   *   The existing EncryptedFieldValue entity.
+   */
+  public function getExistingEntity(ContentEntityInterface $entity, $field_name, $delta, $property);
```

Comment:
Because I changed the method from protected to public, and it's best practice to put public functions in the interface (e.g. for testing purposes)